tests/bluetooth/tester/btp_spec.txt

```diff
@@ -1391,6 +1391,44 @@ Commands and responses:
 
 		In case of an error, the error response will be returned.
 
+	Opcode 0x08 - IV Update test mode
+		Controller Index:	<controller id>
+		Command parameters:	Enable (1 octet)
+
+		This command is used to enable/disable the IV Update test mode.
+
+		In case of an error, the error response will be returned.
+
+	Opcode 0x09 - IV Update toggle state
+		Controller Index:	<controller id>
+		Command parameters:	<none>
+		Response parameters:	State (1 octet)
+
+		Valid State response values:
+					0x00 = Normal
+					0x01 = IV Update in Progress
+
+		This command is used to make a IV Update state transition
+		while in IV Update test mode, ignoring the 96 hour limit.
+
+		In case of an error, the error response will be returned.
+
+	Opcode 0x0a - Network Send
+		Controller Index:	<controller id>
+		Command parameters:	Valid parameters (1 octet)
```

Comment:
What does "Valid parameters (1 octet)" mean?

Reply:
TTL parameter is not always given in PTS MMI, so I needed to some bitfield to indicate which paramaters has to be taken into account